### PR TITLE
restore DID format, add generic URI format

### PR DESCRIPTION
**Pull request by jricher**
Restores the DID format that was removed in -11, and adds a generic "URI" format as well. Also adds normative advice on choosing the most-specific format possible for an identifier.

### CODE REVIEW

**Review by PJ1288 — APPROVED** [review]

**Comment by PJ1288** [comment]
@jricher I will open a new PR incorporating your changes and making updates to the version, date etc. Will also include the XML format. Once done, Annabelle will have to merge it since I don't have write access to this repository.
Thanks!!

**Comment by PJ1288** [comment]
@jricher  Here's the new PR incorporating your changes with updated version and dates. Also got the XML format going.
https://github.com/richanna/secevent/pull/9

**Comment by jricher** [comment]
Replaced by #9

**Pull request closed without merging**
